### ISSUE TITLE
Consolidate tags

### ISSUE THREAD

**Issue by nathanshammah** [unitaryfund/metriq-app#546]
Tags are quite fragmented at the moment, with some duplicates. 
- [x] A. Add guidelines for tags to the User Guidelines page. We decided that tags should be as explicit as possible ('variational quantum algorithm' rather than 'vqa'), single is preferred to plural ('algorithm' rather than 'algorithms').  
- [x] B. Consolidate tags. 

The process of tag consolidation can be done manually by any registered user and can be done with the following steps:
- B.1. Go to the tags page, https://metriq.info/Tags. The Alphabetic listing is best suited for this purpose. 
- B.2. According to the guidelines, add for each submission with the wrong  tag the correct tag and then remove the wrong tag
- B.3. Once step 2 is done for all submission, the view of the tag should disappear from the https://metriq.info/Tags page

**Comment by vprusso** [comment]
@nathanshammah looks like task A is already covered? Or, at least, I think the present phrasing for "tags" on the "user guidelines page" is adequate in terms of description. 

For task B, I'll go through and attempt to consolidate tags.

**Comment by nathanshammah** [comment]
Task B seems completed to me. 
For Task A, @vprusso can you please add to the User Guidelines page, under tags, the fact that  "Tags should be as **explicit** as possible ('variational quantum algorithm' rather than 'vqa') and **single** is preferred to plural ('algorithm' rather than 'algorithms').

**Comment by vprusso** [comment]
@nathanshammah completed in https://github.com/unitaryfund/metriq-app/pull/569
